packages/core/src/external/commonwell/cq-bridge/coverage-enhancer-local.ts

```diff
@@ -46,6 +46,15 @@ export class CoverageEnhancerLocal extends CoverageEnhancer {
       log(`CQ orgs: ${total}, chunks: ${chunks.length}/${chunks.length + fromOrgChunkPos}`);
       log(`patients: ${patientIds.join(", ")}`);
 
+      const originalOrgs =
+        (await this.cwManagementApi.getIncludeList({ oid: orgOID }).catch(error => {
+          // intentionally ignoring the error so we can keep trying to EC
+          log(`ERROR - couldn't revert the list of CQ orgs to the original ones`, error);
```

Comment:
nit: should we perhaps differentiate this message with the other one below? perhaps:

`ERROR - couldn't load original include list of CQ orgs`